app/controllers/posts_controller.rb

```diff
@@ -757,6 +757,8 @@ def create_params
       # We allow `created_at` via the API
       permitted << :created_at
 
+      # We allow `external_id` via the API
```

Comment:
Just curious, do we need to update our API docs to reflect this change?

Reply:
Yep. It's on my list :)